jcommune-model/src/main/resources/org/jtalks/jcommune/migrations/V76__Add_column_Modification_Date and_User_Changed_to_PostComment_Table.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE `post_comment` ADD (
+  `MODIFICATION_DATE` datetime DEFAULT NULL,
+  `USER_CHANGED` BIGINT(20) DEFAULT NULL,
```

Comment:
Number in parenthesis represents how it's displayed in a row, in other words it is display width. So I don't see the reason to use it without some purpose.